README.md

```diff
@@ -71,6 +71,26 @@ The audit is provided as development matures. The latest audit report can be fou
 - Create staking proxy instance on [Launch](https://launch.olas.network/);
 - Vote for staking contracts on [Govern](https://govern.olas.network/).
 
```

Comment:
The checkpoint call arrow should come from ContributeService

Reply:
Why? Checkpoint can be called by anyone. Well, technically the checkpoint is called also during `stake()`, `unstake()` and `claim()`, but that's not connected.

Reply:
Because we intend it to be called by ContributeService and it doesn't make sense to introduce a third role. We certainly don't intend the user to call it.

Reply:
The current graphic is correct but its not helpful when interfacing with Contribute team

Reply:
Ok so let's remove this link or assign it to the `Contribute service`? somebody needs to call `checkpoint()` from time to time.

Reply:
Assign to ContributeService

Reply:
Ot just use `AnyAccount` block